spec/amber/cli/build_spec.cr

```diff
@@ -0,0 +1,62 @@
+require "../../spec_helper"
+require "../../../support/helpers/cli_helper"
+
+include CLIHelper
+
+module Amber::CLI
+  begin
+    describe "building a generated app" do
+      ENV["AMBER_ENV"] = "test"
+      cleanup
+      scaffold_app(TESTING_APP)
+
+      options = ["user:reference", "name:string", "body:text", "age:integer", "published:bool"]
+      temp_options = options - ["user:reference", "age:integer"]
+      MainCommand.run ["generate", "auth", "User"] | (options - ["user:reference"])
+      MainCommand.run ["generate", "scaffold", "Animal"] | temp_options
+      MainCommand.run ["generate", "scaffold", "Post"] | options
+      MainCommand.run ["generate", "scaffold", "PostComment"] | (options + ["post:reference"])
+      MainCommand.run ["generate", "model", "Bat"] | options
+      MainCommand.run ["generate", "migration", "Crocodile"] | options
+      MainCommand.run ["generate", "mailer", "Dinosaur"] | options
+      MainCommand.run ["generate", "socket", "Eagle"] | ["soar", "nest"]
+      MainCommand.run ["generate", "channel", "Falcon"]
+
+      prepare_yaml(Dir.current)
+
+      puts "RUNNING: shard build #{TESTING_APP} - started..."
+      build_result = `shards build #{TEST_APP_NAME}`
+      puts "#{TESTING_APP} build completed..."
+
+      sleep 3000
```

Comment:
I think in crystal the sleep statements are in seconds, not milliseconds.

Reply:
Yes this is intentional while I work on testing the generated app it gives me time too manually run the specs and checks things out

Reply:
I see. You could probably remove the call to `cleanup` and the app will stick around forever. (until you rm -rf it)